controller/JobRoleController.ts

```diff
@@ -28,6 +28,21 @@ module.exports = function(app: Application) {
         } catch (e) {
             console.error(e);
         }
-        res.render('job-role', { jobRole: data, title: 'Job Role' });
+        res.render('job-role', { jobRole: data, title: 'Job Role', isAdmin: req.session.isAdmin });
+    });
+
+    app.post('/job-roles/delete/:id',async (req: Request, res: Response) => {
+        const id: string = req.params.id;
+        let data: JobRoles[] = [];
+
+        try {
+            data = await jobRoleService.getJobRoles(req.session.token);
+            await jobRoleService.deleteJobRoleById(req.session.token, id);
+            data = await jobRoleService.getJobRoles(req.session.token);
```

Comment:
Another option for this, is to just pop the JobRoleId -> Saving a request. However, this would result in the UI being out of sync. Think this is a safer option...

Reply:
Why are you getting the list of job roles again? if you do a page refresh then everything should sync back up